src/Raven.Server/Config/Categories/StorageConfiguration.cs

```diff
@@ -91,6 +91,12 @@ public class StorageConfiguration : ConfigurationCategory
         [ConfigurationEntry("Storage.SyncJournalsCountThreshold", ConfigurationEntryScope.ServerWideOrPerDatabase)]
         public int SyncJournalsCountThreshold { get; set; }
 
+        [Description("Max amount of unsynchronized bytes before requesting a sync and removing unused journal files.")]
+        [DefaultValue(256)]
+        [SizeUnit(SizeUnit.Megabytes)]
+        [ConfigurationEntry("Storage.SyncJournalsMaxSizeInMegabytesThreshold", ConfigurationEntryScope.ServerWideOrPerDatabase)]
```

Comment:
Consistency & conventions: 
- `Storage.SyncJournalsMaxSizeThresholdInMb` for ConfigurationEntry
- `SyncJournalsMaxSizeThreshold` for property name